doc/tutorial/using_gpu.txt

```diff
@@ -470,7 +470,7 @@ the GPU object directly.  The following code is modifed to do just that.
 
   rng = numpy.random.RandomState(22)
   x = shared(numpy.asarray(rng.rand(vlen), config.floatX))
-  f = function([], sandbox.gpuarray.basic_ops.gpu_from_host(tensor.exp(x)))
+  f = function([], sandbox.gpuarray.basic_ops.GpuFromHost(None)(tensor.exp(x)))
```

Comment:
I don't like the idea to force user to always specify the context. I think we should do one of:

1) make gpu_from_host default to None.
2) make gpu_from_host be a special case where it mean any GPU context and then we will keep it on the GPU but it mean the user don't care which context. We should not make it more difficult to use 1 GPU in the new back-end for the purpose that we can support more then 1.

I made an issue about this:

https://github.com/Theano/Theano/issues/4457